packages/cli/src/commands/doctor/runAutomaticFix.ts

```diff
@@ -1,46 +1,49 @@
-// @flow
 import chalk from 'chalk';
-import ora from 'ora';
+import ora, {Ora} from 'ora';
 import {logger} from '@react-native-community/cli-tools';
-// $FlowFixMe - converted to TS
 import {HEALTHCHECK_TYPES} from './healthchecks';
-import type {EnvironmentInfo} from './types';
+import {EnvironmentInfo, HealthCheckCategoryResult} from './types';
 
-const AUTOMATIC_FIX_LEVELS = {
-  ALL_ISSUES: 'ALL_ISSUES',
-  ERRORS: 'ERRORS',
-  WARNINGS: 'WARNINGS',
-};
+export enum AUTOMATIC_FIX_LEVELS {
+  ALL_ISSUES = 'ALL_ISSUES',
+  ERRORS = 'ERRORS',
+  WARNINGS = 'WARNINGS',
+}
 
-export {AUTOMATIC_FIX_LEVELS};
-export default async ({
+interface RunAutomaticFixArgs {
+  healthchecks: HealthCheckCategoryResult[];
+  automaticFixLevel: AUTOMATIC_FIX_LEVELS;
+  stats: {
+    errors: number;
+    warnings: number;
+  };
+  loader: Ora;
+  environmentInfo: EnvironmentInfo;
+}
+
+export default async function({
   healthchecks,
   automaticFixLevel,
   stats,
-  loader,
   environmentInfo,
-}: {
-  healthchecks: any,
-  automaticFixLevel: $Values<typeof AUTOMATIC_FIX_LEVELS>,
-  stats: {errors: any, warnings: any},
-  loader: typeof ora,
-  environmentInfo: EnvironmentInfo,
-}) => {
+}: RunAutomaticFixArgs) {
   // Remove the fix options from screen
-  // $FlowFixMe
-  process.stdout.moveCursor(0, -6);
-  // $FlowFixMe
-  process.stdout.clearScreenDown();
+  if (process.stdout.isTTY) {
+    // @ts-ignore
+    process.stdout.moveCursor(0, -6);
```

Comment:
Those should be part of `readline` as they are apparently not typed under `process.stdout`: DefinitelyTyped/DefinitelyTyped#31505.

I can change those later as there's no harm on ignoring this.